deployment/index.ts

```diff
@@ -188,6 +188,12 @@ const googleConfig = {
   clientSecret: oauthConfig.requireSecret('googleSecret'),
 };
 
+const oktaConfig = {
+  clientId: oauthConfig.requireSecret('oktaClient'),
```

Comment:
if Otka login is not relevant for the SaaS service, I guess this one should be just `getSecret` and not `requireSecret`?

Reply:
I kind of added it here so we still have a way of testing it in prod 🤔 But you are right, we can make it `getSecret` - @kamilkisiela what do you think?